custom/conf/app.ini.sample

```diff
@@ -509,6 +509,10 @@ DELIVER_TIMEOUT = 5
 SKIP_TLS_VERIFY = false
 ; Number of history information in each page
 PAGING_NUM = 10
+; Proxy server URL, support http://, https//, socks://, blank will follow environment http_proxy/https_proxy
+PROXY_URL = 
+; All request domains needed to proxy, 
+PROXY_DOMAINS = 
```

Comment:
would be nice if this would support also wildcards

Reply:
> 
> 
> would be nice if this would support also wildcards

Yes, at least support a single `*`?

Reply:
Will change that.